jOOQ/src/main/java/org/jooq/conf/Settings.java

```diff
@@ -65,6 +67,22 @@
     protected StatementType statementType = StatementType.PREPARED_STATEMENT;
     @XmlElement(defaultValue = "true")
     protected Boolean executeLogging = true;
+    @XmlElement(defaultValue = "DEBUG")
+    protected Log.Level executeLoggingBindVariables = Log.Level.DEBUG;
+    @XmlElement(defaultValue = "false")
+    protected Boolean executeLoggingAbbreviatedBindVariables = false;
```

Comment:
Wrong default. Should be true. (not abbreviating bind variables can explode the logs, e.g. when a bind variable has several mega bytes of LOB data)